packages/chayns-toolkit/src/cli.ts

```diff
@@ -52,4 +53,14 @@ program
 		console.info("")
 	})
 
+program
+	.command("test")
+	.description("tests your code for possible errors using jest")
```

Comment:
Maybe we could change the wording here a little bit. This sounds like we're doing some light tests, but if your test fails (and you've written solid tests), it probably is not a _possible_ error.